src/Commands/Update.php

```diff
@@ -145,6 +152,10 @@ public function execute()
 
         $this->rawOutput = $output['output'];
 
```

Comment:
@bennothommo I am skipping the command output parsing in `dry-run` mode. do you think we should still parse the output and see what packages WOULD be installed/removed/updated ?

Reply:
Yeah I think we'd have to, @mjauvin. That's the entire point of running a dry run 😜

Reply:
> Yeah I think we'd have to, @mjauvin. That's the entire point of running a dry run stuck_out_tongue_winking_eye


@mjauvin looks good to me. Could you just add a unit test for this, making sure that the output is available and also ensuring that no composer.lock file or vendor directory is generated?

Done!

@bennothommo Yes if we want to AUTOMATE the dry-run usage, no if we only want to show the output to the user.